imap_processing/utils.py

```diff
@@ -258,34 +258,42 @@ def packet_file_to_datasets(
 
     with open(packet_file, "rb") as binary_data:
         packet_generator = packet_definition.packet_generator(binary_data)
```

Comment:
You might be able to update this generator instantiation to skip the bad packets:
https://github.com/lasp/space_packet_parser/blob/552701b568750fd68298b66a1f813d18c43ac046/space_packet_parser/definitions.py#L481

```suggestion
        packet_generator = packet_definition.packet_generator(binary_data, parse_bad_pkts=False)
```

Reply:
I tried that and have the same error.